src/transformers/utils/quantization_config.py

```diff
@@ -33,8 +34,96 @@
 logger = logging.get_logger(__name__)
 
 
+class QuantizationMethod(str, Enum):
+    BITS_AND_BYTES = "bitsandbytes"
+    GPTQ = "gtpq"
+
+
+@dataclass
+class QuantizationConfig:
```

Comment:
```suggestion
class QuantizationConfigMixin:
```
Nit: it's a mixin more than a base class.

Reply:
changed !